gluefactory/train.py

```diff
@@ -73,6 +73,7 @@
     "pr_curves": {},
     "plot": None,
     "submodules": [],
+    "use_gpu": True
```

Comment:
what happens if this is true and cuda is not available?

Reply:
Then [this](https://github.com/FelixMoeller3/glue-factory/blob/abccb0e33984a699630310d140e6313352331eac/gluefactory/train.py#L237) assert fails